metricbeat/beater/module_test.go

```diff
@@ -0,0 +1,95 @@
+// +build !integration
+
+package beater_test
```

Comment:
Yes, intentional. This is to ensure no unexported functions are required to make use of the modules (black-box testing).